src/widgets/ot_color_picker.eliom

```diff
@@ -27,23 +27,23 @@ let hsv_to_rgb h s v =
   let h = float_of_int h in
   let c = v *. s in
   let h1 = h /. 60. in
-  let x = c *. (1. -. (abs_float ((mod_float h1 2.) -. 1.))) in
+  let x = c *. (1. -. abs_float (mod_float h1 2. -. 1.)) in
```

Comment:
Can you put all the formatting changes in a separate commit?

Reply:
If you do rebase -i and amend the first commit with ocamlformat on that file, this should remove this kinf of lines.